src/web/layouts/CommentLayout.tsx

```diff
@@ -231,8 +219,11 @@ export const CommentLayout = ({ CAPI, NAV }: Props) => {
                             sectionUrl={CAPI.sectionUrl}
                             guardianBaseURL={CAPI.guardianBaseURL}
                             pillar={CAPI.pillar}
-                            badge={GE2019Badge}
+                            badge={CAPI.badge}
                             inLeftCol={true}
+                            frontendAssetsFullURL={
+                                CAPI.config.frontendAssetsFullURL
```

Comment:
I think we need to stop using `CAPI.config`, in the hope of one day removing it. Currently this variable is present in `guardian.config.frontendAssetsFullURL` idk if there is another way of sourcing it from there instead?

Reply:
Hmm, I think that's just client side? I need to use the assetsUrl server side. There's a lot defined in `CAPI.configType`, while your comment rings a bell, can you remind me why we want to remove it and how we might take the defined fields elsewhere?

Reply:
`guardian.config` is the same as `guardian.app.data.CAPI.config` (from what iv seen) 
I think we should look at removing this duplication, so we should strive to use one or the other not both. Because CAPI is in theory about content, I think it makes sense to remove CAPI.config.

I understand this is part of a larger conversation on CAPI, that likely merits a meeting with the CAPI team, so I am happy to ignore it in the meantime

Reply:
Upon further investigation, you are right `guardian.config` is not accessible on the server, so it makes sense to fetch it this way